docs/developers_guide/landice/api.rst

```diff
@@ -231,6 +231,25 @@ hydro_radial
    visualize.Visualize.run
    visualize.visualize_hydro_radial
 
+ismip6_forcing
+~~~~~~~~~~~~~~
+
+.. currentmodule:: compass.landice.tests.ismip6_forcing
+
+.. autosummary::
+   :toctrr: generate/
+   
+   Ismip6Forcing
+
+   atmosphere.Atmosphere
+   atmosphere.Atmosphere.run
+
+   ocean_basal.OceanBasal
+   ocean_basal.OceanBasal.run
+   
+   ocean_thermal.OceanThermal
+   ocean_thermal.OceanThermal.run
```

Comment:
I think there are several more functions and methods that need to be added to the API.  Let me know if you need help with that.